src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/WebSocketsTransport.cs

```diff
@@ -47,7 +47,7 @@ public WebSocketsTransport(WebSocketOptions options, Channel<byte[]> application
         {
             Debug.Assert(context.WebSockets.IsWebSocketRequest, "Not a websocket request");
 
-            using (var ws = await context.WebSockets.AcceptWebSocketAsync())
+            using (var ws = await context.WebSockets.AcceptWebSocketAsync(_options.SubProtocol))
```

Comment:
Is this going to be used to OPTION-less websocket connections to negotiate hub protocol?

Reply:
Nope. This is for lower level things that use the websocket protocol (like MQTT)